middleware/packet-forward-middleware/packetforward/module_test.go

```diff
@@ -363,74 +363,6 @@ func TestOnRecvPacket_ForwardAmountInt256(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func TestOnRecvPacket_ForwardWithFee(t *testing.T) {
```

Comment:
We no longer check fee, and already unit test without fee above. So this test is useless now